### PR TITLE
Bug 1524321 – Removing truncation + character counts

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

File: content-src/components/DiscoveryStreamComponents/DSCard/DSCard.jsx [hunk 1]
```diff
@@ -31,17 +32,17 @@ export class DSCard extends React.PureComponent {
         </div>
         <div className="meta">
           <div className="info-wrap">
-            <header className="title">{this.props.title}</header>
-            {this.props.excerpt && <p className="excerpt">{this.props.excerpt}</p>}
+            <header className="title">{truncateText(this.props.title, 64)}</header>
```
Comment (k88hudson) [inline content-src/components/DiscoveryStreamComponents/DSCard/DSCard.jsx]: Good refactor :+1:

**Comment by gvn** [comment]
**We may be removing the ellipses and character count, so don't merge for now.**

**Comment by gvn** [comment]
Should be a pretty quick R? for whoever has time...thanks!

**Comment by Mardak** [comment]
This commit got merged without a bug number. @gvn how are you merging these? This has happened before with https://github.com/mozilla/activity-stream/pull/4717#issuecomment-458199600

**Comment by gvn** [comment]
@Mardak I know. It was a mistake.

**Pull request merged**